modules/SofaUserInteraction/src/SofaUserInteraction/SuturePointPerformer.inl

```diff
@@ -84,7 +83,12 @@ void SuturePointPerformer<DataTypes>::start()
         }
         else if (!triangleContainer)
         {
-            msg_error(this->interactor) << "Can't find triangleContainer.";
+            msg_error(this->interactor) << "Can't find a topology.";
```

Comment:
Does it make sense here to set component state to invalid ?

Reply:
maybe but I did not really go further than replacing topology things.
Probably related to what @alxbilger is seeing, about getting a TopologyAccessor (and doing something if not found etc)

Reply:
This is a performer (from mouse operation), I'm not sure it inherites from BaseObject

Reply:
This is a performer (from mouse operation), I'm not sure it inherites from BaseObject